### DISCUSSION TITLE
Allow packages in "includes"

### DISCUSSION THREAD

**Discussion by Faithfinder** [chakra-ui/panda#945] [Ideas]
### Description

I would like to include stuff from component library using package name

### Problem Statement/Justification

Reaching into a package like `'./node_modules/@acme-org/design-system/src/**/*.tsx'` is breaking encapsulation. 

### Proposed Solution or API

I would prefer to specify `@acme-org/design-system/panda-source`.

**Comment by astahmer** [comment]
this solves it https://github.com/chakra-ui/panda/pull/1422
